### PR TITLE
add sphinx boot theme

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@conda-forge/core I think this is ready to go!

File: recipes/sphinx_bootstrap_theme/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "sphinx-bootstrap-theme" %}
+{% set version = "0.4.12" %}
+{% set sha256 = "630401f27ece7a49ae3d8f028cf9e885" %}
```
Comment (ocefpaf) [inline recipes/sphinx_bootstrap_theme/meta.yaml]: This is a `mdf5` and not a `sha256`. That is the trap of all that jinja vars everywhere...

**Comment by jakirkham** [comment]
So we already have this feedstock. It has been here since late June.

I have stopped the conversion of this recipe, but we will need to back it out of staged-recipes.

Please consider adding yourself as a maintainer @CJ-Wright .

cc-ing @conda-forge/sphinx-bootstrap-theme (for awareness)

xref: https://github.com/conda-forge/sphinx-bootstrap-theme-feedstock

**Comment by jakirkham** [comment]
Reverting this addition in PR ( https://github.com/conda-forge/staged-recipes/pull/1883 ).

**Comment by CJ-Wright** [comment]
@jakirkham Sorry about that I thought I checked for this!

**Comment by jakirkham** [comment]
Not a problem. Honestly, I have made this mistake before too.

**Comment by scopatz** [comment]
It probably couldn't be fully robust, but maybe this is something the linter can check for.

**Comment by CJ-Wright** [comment]
That and an existing example/meta.yml file.

**Pull request merged**